shared/common-adapters/copyable-text.js.flow

```diff
@@ -0,0 +1,11 @@
+// @flow
+import {Component} from 'react'
+
+export type Props = {
+  value: string,
+  style?: Object,
+  extras$desktop?: HTMLTextAreaElement,
+  extras$native?: Object,
```

Comment:
This feels a bit hacky, and I think doesn't make sense when you are using this from a file that is common to desktop/native.

How about this:

this file is only for things that are common to desktop and mobile, then you can have `extraStyle` in props in `copyable-text.native`.


Then if something needs to use `extraStyle` in native it has to import `copyable-text.native`. So it enforces the rule that if you are supplying this extra native prop it has to be inside a native file already.